package.json

```diff
@@ -7,10 +7,11 @@
   "react-native": "dist-native/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "tsc --emitDeclarationOnly && BROWSER=1 parcel build src/index.ts",
+    "build": "tsc && BROWSER=1 parcel build src/index.ts",
```

Comment:
Btw do you maybe know how to pass env variables to tsc? Similar to how we pass BROWSER=1 here to parcel.

Reply:
Can you elaborate on this more. What is the case and what do you want to achieve?

Reply:
For example if I want to compile with tsc and set the BROWSER=1 flag

Reply:
Hm, I think I have used env variables like this with tsc - `BROWSER=1 tsc ...`. But that was some time ago. I guess it is not working for Avo Inspector use case?